Obedient-Lobsters/WorkforceManagement
# Employee Edit Implementation

# Description
This adds the ability to edit the lastname, computer assigned, department assigned, and the training programs assigned to an employee. I will be working on updating the README ad well as commenting my code, but I wanted to get this up so you guys could be checking the functionality.  
 Fixes # 10
 ## Type of change
 # Testing Instructions
If you need to create the database:

Start SQL Server Management Studio

Run BangazonDatabaseCreateSeed.sql in SSMS

Update the appsetting.json in Visual Studio community to reflect your desktop and your server

If your database is already seeded or has completed the steps above:

In the terminal, git fetch --all and git checkout wk_EmployeeEdit to switch branches

Start the program by cd'ing into the WorkforceManagement (make sure you are deep enough, the .sln file is a few WorkforceManagement directories deep) and using the command start WorkforceManagement.sln.

Press F5 to start the program in Visual Studio

Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the Employees link in the navbar, and you should see a list with all Employees in the database.

Click on the "Edit" link. To test all functionality:
1. Change the last name of an employee
1. Change the computer assigned to an employee
1. Change the Department assigned to an employee
1. Deselect one of the training programs that were preselected, and then add two more
1. Take not of all of your changes and then click save. You should be redirected to the employee list view. Go either into details or edit again and compare the above items to the ones that you changed them to. If what is shown matches the changes you made, everything should be working correctly.
 # Checklist:
